pyodbc.cpython-311-x86_64-linux-gnu.so,sha256=iHxC2WR-mH82pkuW0n4KBAbyuOl-my5_B5BwrSKexuE,956665
pyodbc.pyi,sha256=rx05N8EYqujKI3FWtMikUjqMPTv4VQg0ChXny9DaisE,32326
pyodbc-5.0.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.0.0.dist-info/WHEEL,sha256=48wUIcZcdQ2pWN7qt0HP02Cvv6HIQZGsSgx3PsepNj8,152
pyodbc-5.0.0.dist-info/RECORD,,
pyodbc-5.0.0.dist-info/METADATA,sha256=UCCQ_QLPO9Mamlo69HQstSc1QzMfiCYTSUJZIm2clXA,2717
pyodbc-5.0.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
